codeship-steps.yml

```diff
@@ -1,8 +1,7 @@
-- type: parallel
-  steps:
-    - name: Python 2 Tests
-      service: python2
-      command: nosetests -sv .
-    - name: Python 3 Tests
-      service: python3
-      command: nosetests -sv .
+- name: test
+  service: indico-client
```

Comment:
can we try and retain our parallel python tests to check 2 and 3? or are we deprecating python 2 support?

Reply:
That is a good question, we absolutely could have a parallel run of the tests using python 2 or multiple versions of python, but are we planning to continue to support python 2 given the sunset date has passed over a year ago?

Reply:
Okay, per slack discussion it sounds like it's safe to drop python 2 support, but I did add in tox for parallel testing of multiple versions of 3, although I'll want to polish that some more